app/src/main/java/com/github/globant/githubsubscribers/subscribersdetail/interactor/SubscriberDetailInteractorImpl.java

```diff
@@ -24,8 +25,12 @@ public void getUserData(String userName, final OnFinishedListener listener) {
         call.enqueue(new Callback<User>() {
             @Override
             public void onResponse(Call<User> call, Response<User> response) {
-                User userItem = response.body();
-                listener.onFinishedUser(userItem);
+                if (response.isSuccessful()) {
+                    User userItem = response.body();
+                    listener.onFinishedUser(userItem);
+                } else {
+                    listener.onFailureUser(Constants.MESSAGE_FAILED_SERVICE);
```

Comment:
This part will be to get better by Edwin, he will implement a best way for manage the errors in the next PR, because he started this task